backend/mobilityAI/Models/Achievement.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using System.Globalization;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace mobilityAI.Models {
+    public class Achievement {
```

Comment:
Maybe a more descriptive name like `ActivityGoals` or `ActivityTargets`, what do you think?